modules/influxdb/src/test/java/org/testcontainers/containers/InfluxDBV1TestImages.java

```diff
@@ -2,6 +2,6 @@
 
 import org.testcontainers.utility.DockerImageName;
 
-public interface InfluxDBTestImages {
+public interface InfluxDBV1TestImages {
```

Comment:
Small point: could we just have one interface to hold both `DockerImageName` constants?

e.g.
```
DockerImageName INFLUXDB_V1_TEST_IMAGE = DockerImageName.parse("influxdb:1.4.3");
DockerImageName INFLUXDB_V2_TEST_IMAGE = DockerImageName.parse("influxdb:2.0.7");

```